docs/source/quick-start.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>This still looks like everything is commented out for some reason. Can you uncomment and then also add a quick markdown cell explaining the interpretation of the confidence intervals vs. blue line here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/raybellwaves/xskillscore/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
We don't have matplotlib installed. Locally I got that error. Although xr has matplotlib.

Reply:
Works now